installer/pkg/common/storage.go

```diff
@@ -47,11 +46,11 @@ func StorageConfig(context *RenderContext) storageconfig.StorageConfig {
 		res = &storageconfig.StorageConfig{
 			Kind: storageconfig.MinIOStorage,
 			MinIOConfig: storageconfig.MinIOConfig{
-				Endpoint:        "minio:9000",
+				Endpoint:        fmt.Sprintf("minio.%s.svc.cluster.local:%d", context.Namespace, MinioServiceAPIPort),
 				AccessKeyID:     context.Values.StorageAccessKey,
 				SecretAccessKey: context.Values.StorageSecretKey,
 				Secure:          false,
-				Region:          context.Config.Metadata.Region,
+				Region:          "local", // Force this value
```

Comment:
Please update the comment to reflect why this happens, rather than what's happening